python/ray/actor.py

```diff
@@ -1167,6 +1167,10 @@ def _actor_method_call(
 
         if num_returns == "dynamic":
             num_returns = -1
+        elif num_returns == "streaming":
+            # TODO(sang): This is a temporary private API.
+            # Remove it when we migrate to the streaming generator.
+            num_returns = -2
```

Comment:
I know this is ugly, but this is the best I could come up with... We can remove it once we deprecate the old implementation of `num_returns="dynamic"`. We have to keep that implementation because until we make it work for lineage reconstruction, we can use the new impl

Reply:
add a constant?